### PR TITLE
1374_fixed

### PR DESCRIPTION
**Pull request by pizzaboi21**
## Description

Replaced the expected annotation with Assert.assertThrows in HealthCheckConfigTest.java for verifying the exceptions.

## Type of change

 - [ ] Bug fix (non-breaking change which fixes an issue)
 - [ ] Feature (non-breaking change which adds functionality)
 - [ ] Breaking change (fix or feature that would cause existing functionality to change
 - [x] Chore (non-breaking change which doesn't affect codebase;
   test, version modification, documentation, etc.)

## Checklist
 - [x] I have read the [contributing guidelines](https://www.eclipse.org/jkube/contributing)
 - [x] I signed-off my commit with a user that has signed the [Eclipse Contributor Agreement](https://www.eclipse.org/legal/ECA.php)
 - [x] I Added [CHANGELOG](../CHANGELOG.md) entry
 - [x] I have implemented unit tests to cover my changes
 - [x] I have updated the [documentation](../kubernetes-maven-plugin/doc) accordingly
 - [x] No new bugs, code smells, etc. in [SonarCloud](https://sonarcloud.io/dashboard?id=jkubeio_jkube) report
 - [x] I tested my code in Kubernetes
 - [x] I tested my code in OpenShift

### CODE REVIEW

File: jkube-kit/config/image/src/test/java/org/eclipse/jkube/kit/config/image/build/HealthCheckConfigTest.java [hunk 1]
```diff
@@ -92,65 +93,80 @@ public void testGoodHealthCheck7() {
...
-                .mode(HealthCheckMode.none)
-                .interval("2s")
-                .build()
-                .validate();
+        Assert.assertThrows(IllegalArgumentException.class, () -> {
+            HealthCheckConfiguration.builder()
+                    .mode(HealthCheckMode.none)
+                    .interval("2s")
+                    .build()
+                    .validate();
```
Comment (rohanKanojia) [inline jkube-kit/config/image/src/test/java/org/eclipse/jkube/kit/config/image/build/HealthCheckConfigTest.java:103]: Move this builder initialization out of assertThrows block (same for others).

```java
    @Test
    public void testBadHealthCheck1() {
        HealthCheckConfiguration healthCheckConfiguration = HealthCheckConfiguration.builder()
                .mode(HealthCheckMode.none)
                .interval("2s")
                .build();

        assertThrows(IllegalArgumentException.class, healthCheckConfiguration::validate);
    }
```

**Comment by manusa** [comment]
Hi @pizzaboi21 ,
Polite ping. Are you still working on this issue?

**Comment by pizzaboi21** [comment]
> Hi @pizzaboi21 , Polite ping. Are you still working on this issue?
Hi @manusa, 
when @rohanKanojia mentioned that I have to move the builder initialization out of the assertThrows block, I was working on that. But in the meanwhile, someone else submitted their pull request regarding this same issue and their PR got merged.
Should I close this PR or still work on it?

**Comment by manusa** [comment]
Oh sorry, I only checked the request changes and missed that a fix for this was already merged.
I'll close the PR.

**Pull request closed without merging**